definitions/npm/mqtt-packet_v6.6.x/flow_v0.84.x-/mqtt-packet_v6.6.x.js

```diff
@@ -0,0 +1,268 @@
+declare module 'mqtt-packet' {
+  declare export type QoS = 0 | 1 | 2;
+
+  declare export type ProtocolVersion = 3 | 4 | 5;
+
+  declare export type PacketCmd =
+    | 'connack'
+    | 'connect'
+    | 'disconnect'
+    | 'pingreq'
+    | 'pingresp'
+    | 'puback'
+    | 'pubcomp'
+    | 'publish'
+    | 'pubrel'
+    | 'pubrec'
+    | 'suback'
+    | 'subscribe'
+    | 'unsuback'
+    | 'unsubscribe';
+
+  declare type CommonPacketFields = {|
+    messageId?: number,
+    length?: number,
+  |};
+
+  declare export type IPacket = {
+    ...CommonPacketFields,
+    cmd: PacketCmd,
+    ...
+  };
+
+  declare export type IConnectPacket = {|
+    ...CommonPacketFields,
+    cmd: 'connect',
+    clientId: string,
+    protocolVersion?: ProtocolVersion,
+    protocolId?: 'MQTT' | 'MQIsdp',
+    clean?: boolean,
+    keepalive?: number,
+    username?: string,
+    password?: Buffer,
+    will?: {|
+      topic: string,
+      payload: Buffer,
+      qos?: QoS,
+      retain?: boolean,
+      properties?: {|
+        willDelayInterval?: number,
+        payloadFormatIndicator?: number,
+        messageExpiryInterval?: number,
+        contentType?: string,
+        responseTopic?: string,
+        correlationData?: Buffer,
+        userProperties?: Object,
```

Comment:
As you probably know `Object` is now aliased to `any`.

Reply:
Whoops, forgot to replace those from the TS defs.  I replaced them with `{ ... }` now